sdks/python/apache_beam/io/sources_test.py

```diff
@@ -53,7 +53,7 @@ def read(self, range_tracker):
       for line in f:
         if not range_tracker.try_claim(current):
           return
-        yield line.rstrip('\n')
+        yield line.decode().rstrip('\n')
```

Comment:
In case you are curious, we changed this to `line.rstrip(b'\n')` to fix the test in Python 3.

Reply:
ah.  good to know! :-)  Thanks!